js_modules/dagit/packages/core/src/nav/InstanceWarningIcon.tsx

```diff
@@ -21,14 +21,14 @@ export const InstanceWarningIcon = React.memo(() => {
 
     // Find any schedules or sensors in the repo list.
     for (const repo of options) {
-      const {schedules, sensors} = repo.repository;
-      if (sensors.length) {
+      if (repo.repository.sensors.some((s) => s.sensorState.status === 'RUNNING')) {
```

Comment:
Works as-is, but if you want to smoosh these (even without es19's flat), lodash can help you do

   ```
 const anySensors = flattenDeep(options.map((o) => o.repository.sensors)).some(
      (s) => s.sensorState.status === 'RUNNING',
    );
    const anySchedules = flattenDeep(options.map((o) => o.repository.schedules)).some(
      (s) => s.scheduleState.status === 'RUNNING',
    );
```